packages/dd-trace/src/git_properties.js

```diff
@@ -1,6 +1,20 @@
 const commitSHARegex = /git\.commit\.sha=([a-f\d]{40})/
 const repositoryUrlRegex = /git\.repository_url=([\w\d:@/.-]+)/
 
+function removeUserSensitiveInfo (repositoryUrl) {
+  try {
+    // repository URLs can contain username and password, so we want to filter those out
+    const parsedUrl = new URL(repositoryUrl)
+    if (parsedUrl.password) {
```

Comment:
You are removing username and password in the next line, but only if password is present. You are not removing the username if password is not present:

`https://username@github.com` is not going to be modified, but `https://username:password@github.com` will be modified to `https://github.com` instead of `https://username@github.com`
If username is sensitive, I guess that we should check the username or password here.

```suggestion
    if (parsedUrl.username || parsedUrl.password) {
```